modules/home/theme/gruvbox/default.nix

```
@@ -103,31 +103,32 @@ in {
           yellow   = "#d79921";
 
           in {
-            bars = [{
-              position = "top";
-
-              colors = {
-                background = bg;
-                statusline = yellow;
-                separator = red;
-
-                focusedWorkspace = {
-                  border = aqua; background = aqua; text = darkgray;
-                };
-
-                activeWorkspace = {
-                  border = darkgray; background = darkgray; text = yellow;
-                };
-
-                inactiveWorkspace = {
-                  border = darkgray; background = darkgray; text = yellow;
+            bars = if (config.mine.workstation.i3.bar.engine == "i3bar") then
```

```suggestion
            bars = optionals (config.mine.workstation.i3.bar.engine == "i3bar")
```

and remove ` else []` from the bottom.

See the implementation/documentation of `lib.optionals` ->
https://github.com/NixOS/nixpkgs/blob/2932be89af72430888ca4bc5b549bc467f218549/lib/lists.nix#L243-L257